README.md

```diff
@@ -129,15 +129,15 @@ For the time being, please consider the following actions:
 4. Remove the containers related to AI
 5. Cleanup your local clone of the recipes: `$HOME/podman-desktop/ai-lab`
 
-### Providing a custom catalog
+### 📖 Providing a custom catalog
 
-The extension provides a default catalog, but you can build your own catalog by creating a file `$HOME/podman-desktop/ai-lab/catalog.json`.
+The extension provides by default a curated list of recipes, models and categories. However, this system is extensible and you can define your own.
 
-The catalog provides lists of categories, recipes, and models.
+To enhance the existing catalog, you can create a file located in the extension storage folder `$HOME/.local/share/containers/podman-desktop/extensions-storage/redhat.ai-lab/user-catalog.json` in the extension storage folder.
```

Comment:
You repeated twice `in the extension storage folder`. One of the two can be removed

Reply:
Thanks, fixed